JobBot-server/jobbot/jobbot/src/test/java/org/dizzybot/jobbot/entities/JobTest.java

```diff
@@ -0,0 +1,52 @@
+package org.dizzybot.jobbot.entities;
+
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JobTest {
+
+    private User user;
+
+    private List<Job> jobAccepted;
+    private List<Job> jobPosted;
+
+    private Job jobTest;
+
+    private Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+
+    @Before
+    public void beforeEach() {
+        this.user = new User("Dizzy Bot", "Password", "dizzybot@email.com");
+        this.jobAccepted = this.user.jobAccepted;
+        this.jobPosted = this.user.jobPosted;
+    }
+
+    @After
+    public void afterEach() {
+        this.jobAccepted = null;
+        this.jobPosted = null;
+    }
+
+    @Test
+    public void testUserJobListsCreateByUser() {
+        assertNotNull(this.user.jobAccepted);
+        assertNotNull(this.user.jobPosted);
+    }
```

Comment:
Do not directly access attribute, access attribute using getter.

Reply:
this is to test the set/get function or is it better to access attributes this way~